Example/Swiftilities.xcodeproj/project.pbxproj

```diff
@@ -313,6 +313,14 @@
 			path = "About View";
 			sourceTree = "<group>";
 		};
+		D98569C923E8C54D007EF367 /* CoreDataTests */ = {
```

Comment:
#nth might be nice to call out the re-org of this structure in the test file itself or something so that future folks can figure out why it's excluded and/or figure out how to port it to the spm tests.

Reply:
Ya good call. Unfortunately you can't ship resources with SPM yet, but it's on the way: https://github.com/apple/swift-evolution/blob/master/proposals/0271-package-manager-resources.md